src/pages/CubePlaytestPage.js

```diff
@@ -292,12 +321,58 @@ const CubePlaytestPage = ({ cube, cubeID, canEdit, decks, draftFormats }) => {
     [addAlert, cubeID, formats],
   );
 
+  const handleSetDefaultFormat = useCallback(
+    async (event) => {
+      const formatIndex = parseInt(event.target.getAttribute('data-index'), 10);
+      try {
+        const response = await csrfFetch(`/cube/defaultdraftformat/${cubeID};${formatIndex}`, {
```

Comment:
I prefer cube/cubeID/defaultdraftformat/formatIndex  but realize that is a bit different than standard. I really don't like the `;` so happy if that gets removed.